packages/vitest/src/defaults.ts

```diff
@@ -1,8 +1,7 @@
 import type { BenchmarkUserOptions, ResolvedCoverageOptions, UserConfig } from './types'
 
 export const defaultInclude = ['**/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}']
-export const defaultExclude = ['**/node_modules/**', '**/dist/**', '**/cypress/**', '**/.{idea,git,cache,output,temp}/**']
-
+export const defaultExclude = ['**/node_modules/**', '**/dist/**', '**/cypress/**', '**/.{idea,git,cache,output,temp}/**', 'vite.config.*', 'vitest.config.*']
```

Comment:
```suggestion
export const defaultExclude = ['**/node_modules/**', '**/dist/**', '**/cypress/**', '**/.{idea,git,cache,output,temp}/**', '{vite,vitest}.config.*']
```
I think we can actually add more configs, like in `defaultCoverageExcludes`

Reply:
Done, I also added it to the docs since it was missing